aries-site/src/pages/templates/lists.mdx

```diff
@@ -65,7 +65,7 @@ Use hover states to indicate which kind of selection is being used. If the entir
 ### Action Menu in List Item
 
 - If space is limited, then don’t use an Action Menu in a List item (repetition of the Action Menu icon uses a lot of space)
-- If items can be selected using only the List contents, then use an Action Menu in a List item. Otherwise, go to a detail view to see additional information for the List item and use an Action Menu there.
+- If more context is needed to perform an action, the entire list row should be clickable and should route the user to a detail page for the List item. On this detail page, an Action Menu can be provided to allow the user to engage with the data.
```

Comment:
Im not best at grammar but could it be 
route the user to a detailed page ? or is detail page the correct way ?

Reply:
Good comment. I think it could go either way, I think the way I was using it was almost more "naming" this type of page.

Reply:
oh okay yeah then! this is good by me